### PR TITLE
[DROOLS-3642] Showing custom "loading" popup on new asset creation

### PR DESCRIPTION
**Pull request by gitgabrio**
@kkufova @Rikkola @danielezonca 

See  https://issues.jboss.org/browse/DROOLS-3642

loadingpopup

### CODE REVIEW

**Comment by Rikkola** [comment]
Not going to tell you how long I watched that picture and waited for it to move...

**Comment by gitgabrio** [comment]
@kkufova 
_Fixed_

File: drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/resources/images/ScenarioSimulationEditorImageResources.java [hunk 1]
```diff
@@ -25,4 +25,7 @@
 
     @Source("BPM_FileIcons_ScenarioSimulation.png")
     ImageResource typeScenarioSimulation();
+
+    @Source("loading.gif")
```
Comment (kkufova) [inline drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/resources/images/ScenarioSimulationEditorImageResources.java]: Do we still need this?

**Comment by gitgabrio** [comment]
@kkufova 
Removed unneeded gif and reference.
:+1: 

Going to merge as soon as the Jenkins build is green.

**Comment by gitgabrio** [comment]
@manstis
Hi! Could you please merge this, squashing commits? Thanks!

**Pull request merged**